analytics/README.md

```diff
@@ -24,3 +24,15 @@ To collect analytics from a subset of found repositories (for testing purposes),
 
 To collect analytics from all found repositories, run:
 ```node run-analytics.js```
+
+## Bonus content
+The bash script file `find-hds-react-version-usage.sh` calls Github API and finds all `hds-react` occurrences in code under the City of Helsinki organisation and gets the corresponding repository name, Github URL and hds-react version and puts them in a JSON file. This could possibly be merged into the analytics scripts to provide even more refined analytics data.
+
+### How to use it
+You need bash and a Github account.
+
+The script uses an environment variable called `GITHUB_TOKEN` which is Personal Access Token (PAT for short) which you can obtain from your [Github settings](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). If you can't be bothered, you can replace the `GITHUB_TOKEN` in the script with your PAT.
```

Comment:
We could maybe add that what scopes are required for the personal access token:

<img width="886" alt="image" src="https://user-images.githubusercontent.com/2777633/191975505-e9f8c903-5d8c-4101-8794-76a50984e2b4.png">

Reply:
A very good point, I'll add it in the comments!